web/src/Tooltip.tsx

```diff
@@ -55,22 +64,61 @@ const InfoIcon = styled(InfoSvg)`
     fill: ${Color.gray6};
   }
 `
+const DismissButton = styled(InstrumentedButton)`
+  ${mixinResetButtonStyle};
+
+  .MuiButton-label {
+    font-size: ${FontSize.smallester};
+    font-style: italic;
+    color: ${Color.grayLight};
+  }
+`
 
 export interface InfoTooltipProps {
+  dismissId?: string // If set, this tooltip will be dismissable, keyed by this string
   displayShadow?: boolean
   idForIcon?: string // Use to semantically associate the tooltip with another element through `aria-describedby` or `aria-labelledby`
 }
 
 export function TiltInfoTooltip(
   props: Omit<TooltipProps, "children"> & InfoTooltipProps
 ) {
-  const { displayShadow, idForIcon, ...tooltipProps } = props
+  const { displayShadow, idForIcon, title, dismissId, ...tooltipProps } = props
   const shadowClass = displayShadow ? "shadow" : ""
 
+  const [
+    dismissed,
+    setDismissed,
+  ] = usePersistentState(`tooltip-dismissed-${props.dismissId}`, false, {
+    keyedByTiltfile: false,
+  })
+  if (dismissed) {
+    return null
+  }
+
+  let content = title
+
+  if (dismissId !== undefined) {
+    content = (
+      <>
+        <div>{title}</div>
+        <DismissButton
+          size="small"
+          analyticsName={"ui.web.dismissTooltip"}
+          analyticsTags={{ tooltip: props.dismissId }}
+          onClick={() => setDismissed(true)}
+        >
+          Hide this info button
```

Comment:
This is a bit longer than some of my earlier iterations (like "dismiss" or "hide this"). Since this is non-undoable action, I think it's extra important the user understands what clicking the button will do.  Happy to hear alternatives (or cheap ways to allow undo!)

Reply:
wdyt about `Don't show again`? or perhaps `Don't show tip again`? i think that "don't show" language is kinda standard for software tutorials and users will understand what it means!

Reply:
good!